python/cuml/_thirdparty/sklearn/preprocessing/_discretization.py

```diff
@@ -199,6 +199,10 @@ def fit(self, X, y=None):
             elif self.strategy == 'quantile':
                 quantiles = np.linspace(0, 100, n_bins[jj] + 1)
                 bin_edges[jj] = np.asarray(np.percentile(column, quantiles))
+                # Workaround for https://github.com/cupy/cupy/issues/4451
```

Comment:
Do we have a issue in our repo to track it?

Reply:
Just added #3317. Thanks for the reminder!